_docs/concepts/policy-and-control/mixer.md

```diff
@@ -63,19 +63,13 @@ to target new or custom infrastructure backends.
 
 ## Configuration state
 
-Mixer's core runtime methods (`Check`, `Report`, and `Quota`) all accept a set of attributes on input and
-produce a set of attributes on output. The work that the individual methods perform is dictated by the set of input
-attributes, as well as by Mixer's current configuration. To that end, the service operator is responsible
-for:
+Mixer's core runtime methods ([Check](https://github.com/istio/api/blob/master/mixer/v1/service.proto#L43) and [Report](https://github.com/istio/api/blob/master/mixer/v1/service.proto#L48)) accept a set of attributes on input. Mixer's current configuration dictates the work that the individual methods perform with the set of input attributes. To that end, the service operator is responsible for:
 
-- Configuring the set of *aspects* that the deployment uses. An aspect is essentially a chunk of configuration
-state that configures an adapter (adapters being binary plugins as described [below](#adapters)).
+- Configuring a set of *handlers* for Mixer-generated data. Handlers are configured adapters (adapters being binary plugins as described [below](#adapters)). An example of handler configuration would be providing a statsd adapter with an IP address for a statsd backend.
 
-- Establishing the types of adapter parameters that Mixer can manipulate. These
-types are described in configuration through a set of *descriptors* (as described [here](./mixer-config#descriptors))
+- Configuring a set of *instances* for Mixer to generate based on attributes and literal values. Example of instances are _metric values_ and _log entries_. They represent a chunk of data that adapter code will operate on. For instance, an operator may configure Mixer to generate `request_count` metric values from attributes such as `destination.service` and `response.code`.
 
-- Creating rules to map the attributes of every incoming request into a 
-specific set of aspects and adapter parameters.
+- Configuring a set of *actions* that provide Mixer with the when, what, how, and where of instance handling. Actions dictate under what conditions instances should be generated, what instances to generated for those conditions and how they should be generated, and which handler should be sent the generated instances. These actions are grouped into *rules*. For instance, an action might tell Mixer to send generated `request_count` metric values to a statsd handler for all Report calls.
```

Comment:
the mixer

Reply:
Actions define the conditions under which instances should be generated, the set of instances to generate, how the instances should be generated, and the handlers that should process the generated instances.

Reply:
"action might tell Mixer" => the Mixer

Reply:
send generated `request_count` metric values -> send generated `request_count` metric instances ?

Reply:
Over all, just as feedback: the first sentence is too abstract, and tad bit philosophical. If you are going to use an example, might be better to relate to each of the when, what, how and where explicitly.

Reply:
I've tried to simplify this a bit and reduce the abstract. Hopefully, I've improved it.